components/main-chef-wrapper/main.go

```diff
@@ -42,6 +47,34 @@ func createDotChef() {
 	os.Mkdir(path, 0700)
 }
 
+func createRubyEnv(){
+//	get installation path
+//	homepath, err := homedir.Dir()
+//	if err != nil {
+//		log.Fatalf(err.Error())
+//	}
+//	todo ==> incase home directory is needed we can add it to string
+	installationPath :=  platform_lib.ExpectedOmnibusRoot() + "/ruby-env.json"
```

Comment:
I think we have to resolve and use `~/.chef-workstation/ruby-env.json`. The operator usually won't be running `chef` with root permissions, so creating this file would fail.

Reply:
resolved https://github.com/chef/chef-workstation/pull/2430/commits/01e46e7fc070e67e3e62b8d01c9e1b3edb1c353b